docs/user-guides/pallet-proxy.md

```diff
@@ -0,0 +1,191 @@
+
+9. **Submit the transaction:** Once you have filled in all the necessary details, find the button to submit the transaction. Click on it to initiate the process.
+
+![33](img/33.png)
+
+You will then be prompted to authorize and sign the transaction. Go ahead and click **Sign and Submit** to create the proxy relationship.
+
+![34](img/34.png)
+
+Once the transaction has been successfully submitted, you will receive some notifications confirming the transaction. 
+
+You can also find the event `proxy.ProxyAdded` in recently emitted events inside  **Network** > **Explorer** tab.
+
+![35](img/35.png)
+
+**Verifying Proxy Account**
+---
+There are many ways of verifying if your proxy was added or not. Easiest way to do so is using the **Accounts** page.
+
+1. Navigate to the Accounts page by clicking on **Accounts** tab and then selecting **Accounts**.
+
+2. Here find you **Primary Account** and click on the 3 dots as seen in the provided picture.
+
+
+![36](img/36.png)
+
+3. Select **Manage proxies** option.
+
+
+Here you can see the list of all proxies that you account has. For this tutorial, it is only **Balances** proxy that we added in the above section.
+
+![37](img/37.png)
+
+:::tip
+
+You can also remove the proxy by clicking on the (X) icon next to the proxy account (in our case **BOB**). After clicking (X) button, the proxy will diappear from the list, Click on `Submit`.
+
+Once the transaction has successfully been submitted, you can review your current proxies or if you removed all proxies you will notice the proxy icon is no longer being displayed next to the primary account.
+
+:::
+
+**Executing a Proxy Transaction**
+---
+ To execute a proxy transaction, go back to the **Extrinsic** page and do the following:
+
+ ## Submitting a Proxy Transaction
+
+To submit a proxy transaction, follow these steps:
+
+1. **Select the proxy account:** Choose the proxy account to submit the transaction from using the "Select Account" dropdown.
+
+2. **Submit the following extrinsic:** From the "Submit the following extrinsic" menu, select "proxy".
+
+3. **Choose the proxy extrinsic:** Select the "proxy" extrinsic.
+
+4. **Select the primary account:** From the "real" dropdown, select **Id** and then select the **Primary Account** (ALICE in our case)
+
+5. Select the **balances** call
+
+6. Choose the **transfer** extrinsic
+
+7. **Enter the destination address:** In the "dest" field, enter the address where you want to send the funds.
+
+8. **Enter the value:** In the "value" field, enter the amount of funds to send.
+
+9. **Click Submit Transaction:** Once you have entered all the necessary details, click on "Submit Transaction" to initiate the transaction.
+
+![38](img/38.png)
+
+Congratulations! You've completed the entire process successfully. You have created a proxy account, reviewed all the proxy accounts linked to your primary account, performed a proxy transaction, and even removed a proxy account. Well done!
```

Comment:
Suggestion: add this video as external resource that covers pure proxy from use case
https://youtu.be/MqY6yU-YLO0

Reply:
thanks for the suggestion.